scripts/seed-display.ts

```diff
@@ -112,13 +120,13 @@ async function init(): Promise<void> {
  *
  * @returns - The user seed as bytes.
  */
-async function getRootSeed(): Promise<Uint8Array> {
+async function getRootSeedAndEmail(): Promise<[Uint8Array, string | null]> {
   const checkInterval = 100;
 
   return new Promise((resolve) => {
     const checkFunc = () => {
-      if (readySeed !== null) {
-        resolve(readySeed);
+      if (readySeed) {
+        resolve([readySeed, readyEmail]);
```

Comment:
`readyEmail` will be `null` on login.

Reply:
I don't know what the flow is for login yet. I figure I'll have to check for the stored JWT on MySky and show a new page if it is not found. But if the email is optional, then if someone doesn't want to enter it, they will get that page every time they login.

Also, I think I'm still waiting on the MySky kernal in order to get saved portal preferences / email.

Lots of open questions here, still.

Reply:
The email is not optional. This is clearly defined in the spec (https://hackmd.io/xk1yHascReiCj0TcKFgegw) which states that `Portal accounts no longer use email/password, they now use email/mysky` and also lists `email` as a parameter for registration and login without specifying that it's optional.

Reply:
As per the call, connecting to the portal account will be optional for now, but the user will be asked for their email every time they sign in until it is provided. The flow can be changed or improved in the future. If the email is not provided, then their MySky account is simply not connected to the portal account.